README.md

```diff
@@ -64,9 +53,9 @@ $ cd RxAndroid/
 $ ./gradlew build
 ```
 
-Futher details on building can be found on the RxJava [Getting Started](https://github.com/ReactiveX/RxJava/wiki/Getting-Started) page of the wiki.
+Futher details on building can be found on the RxJava [Getting Started][start] page of the wiki.
 
-<a href='https://travis-ci.org/ReactiveX/RxAndroid/builds'><img src='https://travis-ci.org/ReactiveX/RxAndroid.svg?branch=0.x'></a>
+<a href='https://travis-ci.org/ReactiveX/RxAndroid/builds'><img src='https://travis-ci.org/ReactiveX/RxAndroid.svg?branch=master'></a>
 
 
 # Sample usage
```

Comment:
I think we could remove "sample usage" since we have a sample app now that demonstrates all the capabilities of RxAndroid (where "all" is actually not that much, hah).